hawkular-wildfly-agent-feature-pack/src/main/resources/modules/system/layers/base/org/hawkular/agent/main/module.xml

```diff
@@ -83,6 +83,7 @@
     <module name="org.jboss.vfs"/>
 
     <!-- other dependencies we need -->
+    <module name="javaee.api"/> <!-- bus-commons -->
```

Comment:
That might not be a bad idea. Split out BasicMessage and its related classes, with all the JSON processing that goes with it, into a smaller commons library.

Agent does NOT use JMS. Never did. The only time it interfaced with the bus was many releases ago when it went through the REST api that sat in front of ActiveMQ. But that was solely a temporary solution and has gone away now. But it never used JMS classes directly.